src/types/app_config_test.go

```diff
@@ -11,6 +11,33 @@ import (
 var _ = Describe("AppConfig", func() {
 	var appConfig types.AppConfig
 
+	var _ = Describe("ValidateProductionFields", func() {
+		It("should throw an error when AppConfig is missing fields", func() {
+			appConfig = types.AppConfig{
+				Production: map[string]string{
+					"url":        "originate.com",
+					"account-id": "123",
+					"region":     "us-west-2",
+				},
+			}
+			err := appConfig.ValidateProductionFields()
+			Expect(err).To(HaveOccurred())
```

Comment:
Can we also test what the error message? And maybe add an test case for when there is no production section?